### DISCUSSION TITLE
Is it possible to add users of GitHub Enterprise Cloud to the repositories of a team level subscription

### DISCUSSION THREAD

**Discussion by anijayn** [community/community#62076] [Enterprise]
### Select Topic Area

Question

### Body

Our team is collaborating with a team in another division of our company. We have a GitHub Enterprise Cloud setup, and the other team has a team-level subscription. They have been trying to provide us access to their repositories, but they are only able to provide access to our personal public GitHub accounts. This is not preferred due to security reasons. So is there a way to somehow use our GHEC accounts instead to access their repos?

**Comment by ahmetpalavan** [comment]
Yes, your GitHub Enterprise Cloud (GHEC) installation has a solution to access other team's repositories. You can use "Organizations" to access other team's repositories using your GHEC accounts.

GitHub Organizations are private spaces where users and teams can come together to manage and share projects. This means that by creating an Organization in your company's GHEC installation, the other team can be invited and thus have access to the other team's repositories.

Here are the steps

Create an Organization from your GHEC accounts in your organization and invite the other team to this Organization.

Users from the other team can join your Organization and create their own repositories under this Organization.

The other team's access to your repositories can be managed at the Organization level. So you can grant or restrict certain permissions in accordance with security policies.

Organizations allow you to easily manage internal collaboration and security requirements. This way, you get a more secure way to access other team's repositories without having to use personal public GitHub accounts.

Remember that managing access and permissions to Organizations should be done by the Organization's owner or administrator. This way, the other team can have access controls in place that suit their needs while ensuring their security.
